lib/shoulda/matchers/action_controller/render_with_layout_matcher.rb

```diff
@@ -64,11 +64,11 @@ def render_with_layout(expected_layout = nil)
       # @private
       class RenderWithLayoutMatcher
         def initialize(expected_layout)
-          if expected_layout
-            @expected_layout = expected_layout.to_s
-          else
-            @expected_layout = nil
-          end
+          @expected_layout = if expected_layout
```

Comment:
Yes it can enforce it and even its the default [Layout/MultilineAssignmentLayout](https://docs.rubocop.org/rubocop/cops_layout.html#layoutmultilineassignmentlayout). I;m not sure why it didn;t compliant, might be a bug ? Will see to it and will report an issue/PR in rubocop if it is.